libs/src/services/discoveryProvider/DiscoveryProviderSelection.test.js

```diff
@@ -202,6 +206,50 @@ describe('DiscoveryProviderSelection', () => {
     assert.strictEqual(s.isInRegressedMode(), true)
   })
 
+  it('can select the discprov that is the least number of blocks behind for past versions', async () => {
+    const behind100 = 'https://behind100.audius.co'
+    nock(behind100)
+      .get('/health_check')
```

Comment:
just to clarify my understanding - health_check always returns 200 even if behind so there's no chance of a 500/other status code from the 200 diff endpoint. this confirms we select the _least_ behind endpoint

Reply:
yep -- the health check by default doesn't return a 500 unless you pass enforce_block_difference